src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    White,
+    Blue,
```

Comment:
```suggestion
    WHITE,
    BLUE,
```
enum values should be constants